src/Polar.cpp

```diff
@@ -181,7 +181,7 @@ Polar::Polar()
 
 #define MAX_WINDSPEEDS_IN_TABLE 200
 #define MESSAGE(S) (S + wxString(_T("\n")) + wxString::FromUTF8(filename) \
-                    + (line > 0 ? (_(" line ") + wxString::Format(_T("%d"), linenum)) : _T("")))
+                    + (line > (void *)0 ? (_(" line ") + wxString::Format(_T("%d"), linenum)) : _T("")))
```

Comment:
Hi,
IMO it's a thinko or a copy and paste error, testing line doesn't make sense.

Should be linenum > 0

Regards
Didier